extensions/amp-story/1.0/story-analytics.js

```diff
@@ -127,23 +129,25 @@ export class StoryAnalyticsService {
 
   /**
    * @param {!StoryAnalyticsEvent} eventType
+   * @param {?Element} opt_element
    */
-  triggerEvent(eventType) {
+  triggerEvent(eventType, opt_element = null) {
```

Comment:
Nit: you don't need the `opt_` prefix with default params

Reply:
Done.